packages/renderer/src/lib/onboarding/Onboarding.svelte

```diff
@@ -1,3 +1,22 @@
+<style>
+#stepBody::-webkit-scrollbar {
+  width: 1em;
+}
+#stepBody::-webkit-scrollbar-track {
+  -webkit-box-shadow: inset 0 0 6px rgba(0, 0, 0, 0.3);
+}
+#stepBody::-webkit-scrollbar-thumb {
+  background-color: #5c5c5c;
+}
+#stepBody::-webkit-scrollbar-thumb:hover {
+  background-color: #707073;
+}
```

Comment:
When I screenshot and eyedrop the scrollbar color values I get #5c5c5c for the lighter moving piece and #34343b for the darker track background.

The colors the rest of the app has for these is #676767 for the lighter moving piece and #424242 for the darker track background. (The closest palette colors are #676767=>charcoal-100 and #424242=>charcoal-400 but those are approximately similar and not exactly the same hex codes.)

I would change to the hex values the app is already using just so the scrollbar doesn't stand out as being different (i think it strikes me as being a bit darker or somehow a bit off? which is why i checked with the eyedropper to see what the values are.)

Reply:
When i tried using charcoal-400 it looked weird so i went to look the exact color of the scrollbar in the troubleshooting. My newbie way to take them was to screenshot the scrollbar in all its state (normal, hover, active) and using the palette. I got these colors:
normal: #676767
hover: #7b7b7b
active: a1a1a1

Following your method i picked these
normal: charcoal-100 (#707073)
hover: charcoal-50 (#767676)
active: gray-700 (#aaabac)

The result/comparison is this one. WDYT?
![scrollbar_color](https://github.com/containers/podman-desktop/assets/49404737/42263936-3826-4442-9c89-8389c6d23b84)

Reply:
Ah you are right that looks weird... I'd go back to the ones you eyedropped!

Reply:
(or I can do some picks from the palette in a mockup to test to make sure it doesn't look weird...)